alfresco-transform-tika/alfresco-transform-tika/pom.xml

```diff
@@ -49,6 +49,10 @@
                     <groupId>org.quartz-scheduler</groupId>
                     <artifactId>quartz</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Why is this change required now (but not previously) ? Are these libraries provided elsewhere ? If not, when does Tika use these libraries at runtime ?